module/VuFind/src/VuFind/Search/EPF/Options.php

```diff
@@ -87,26 +89,16 @@ public function getSearchAction()
         return 'epf-search';
     }
 
-    /**
-     * Return the view associated with this configuration
-     *
-     * @return string
-     */
-    public function getView()
-    {
-        return $this->defaultView;
-    }
-
-    /**
-     * Return the view associated with this configuration
-     *
-     * @return string
-     */
-    public function getEpfView()
```

Comment:
Getting rid of the public getEdsView and getEpfView methods is a bit of a BC break, though I think it's very unlikely that people have customized these things. I'll add a TODO note so we remember to add this to the changelog when merging.